src/applications/disability-benefits/996/constants.js

```diff
@@ -21,7 +21,7 @@ export const LEGACY_APPEALS_URL = '/decision-reviews/legacy-appeals/';
 
 // 8622 is the ID of the <li> wrapping the "Find addresses for other benefit
 // types" accordion
-export const BENEFIT_OFFICES_URL = `${HLR_INFO_URL}#find-addresses`;
+export const BENEFIT_OFFICES_URL = `${HLR_INFO_URL}#find-addresses-for-other-benef-8622`;
```

Comment:
Kinda weird to be so verbose and then chop off `benefit` in the ID :-D

Should the comment above this be updated?

Reply:
Ha ha, oops. thanks!